README.adoc

```diff
@@ -62,7 +62,7 @@ ifndef::ProductRelease,EAPXPRelease[]
 :productImageVersion: 31.0
 :githubRepoUrl: https://github.com/wildfly/quickstart/
 :githubRepoCodeUrl: https://github.com/wildfly/quickstart.git
-:WildFlyQuickStartRepoTag: 31.0.0.Final
+:WildFlyQuickStartRepoTag: 31.0.0.Final-glow-preview
```

Comment:
Please note that any change in README.adoc that was not auto done by the asciidoc-reducer should be moved to related README-source.adoc . I think you should confirm this, and revert all README.adoc

I guess those portedToGlow should now be removed and the shared-docs include file should be updated, wrt the XML snippet change for both server provisioning and bootable jar